src/swap_utils.h

```diff
@@ -19,4 +19,8 @@
 
 #include <stdint.h>
 
-bool parse_swap_config(const uint8_t* config, uint8_t config_len, char* ticker, uint8_t* decimals);
+bool parse_swap_config(const uint8_t* config,
+                       uint8_t config_len,
+                       char* ticker,
+                       uint8_t* decimals,
```

Comment:
Not related, but I don't understand why sometimes the linter wants `int *var` and sometimes `int* var` ...

Reply:
That is a good question.